appOPHD/UI/DetailMap.cpp

```diff
@@ -167,6 +167,18 @@ void DetailMap::drawGrid() const
 }
 
 
+uint8_t DetailMap::glow() const
+{
+	constexpr int glowAmplitude = 120;
+	constexpr int glowOffset = 120;
+	constexpr int throbCycleTime = 2000;
+
+	int sawtooth = static_cast<int>(throbTimer.tick()) % throbCycleTime;
+	int triangle = sawtooth < throbCycleTime / 2 ? sawtooth : (throbCycleTime - sawtooth);
+	return static_cast<uint8_t>(triangle * glowAmplitude / (throbCycleTime  / 2) + glowOffset);
+}
```

Comment:
This function doesn't access member variables for `DetailMap`, so could either be made `static`, or moved out of the class, probably to the unnamed namespace. A `static` method requires a line in the header file, which can trigger recompiles of multiple files if it ever changes. A function in an unnamed namespace will only ever affect the file it's in. For that reason, I tend to prefer functions in unnamed namespaces over `static` member functions.